src/App.jsx

```diff
@@ -14,57 +16,58 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setselectedGood] = useState('Jam');
```

Comment:
The function `setselectedGood` should be renamed to `setSelectedGood` to follow the camelCase naming convention, which is standard for React hooks and JavaScript in general.

Reply:
agree